lib/key.js

```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 const util = require('util');
-const crypto = require('crypto');
-const eccrypto = require('eccrypto');
-const hash = require('hash.js');
+
+const EC = require('elliptic').ec;
+const ec = new EC('secp256k1');
 
 function Key (init) {
```

Comment:
might be useful to let the user initialize a Key instance with pre-known public/private keys, that way you don't just create a new pair every time. this way if you only know the public key you can verify signatures, and if you know both keys you can sign messages